### ISSUE TITLE
AttributeError: 'filter' object has no attribute 'remove'

### ISSUE THREAD

**Issue by mkaito** [afewmail/afew#158]
This has been happening for about a week now. No changes made to config when it happened.

```
Traceback (most recent call last):
  File "/usr/bin/afew", line 11, in <module>
    load_entry_point('afew==1.1.1.dev15+g7f942f3', 'console_scripts', 'afew')()
  File "/usr/lib/python3.6/site-packages/afew/commands.py", line 131, in main
    query_string = get_notmuch_new_query()
  File "/usr/lib/python3.6/site-packages/afew/NotmuchSettings.py", line 41, in get_notmuch_new_query
    return '(%s)' % ' AND '.join('tag:%s' % tag for tag in get_notmuch_new_tags())
  File "/usr/lib/python3.6/site-packages/afew/NotmuchSettings.py", line 35, in get_notmuch_new_tags
    tags.remove("unread")
AttributeError: 'filter' object has no attribute 'remove'

```

```
❯ afew --version  
1.1.1.dev15+g7f942f3

❯ python --version                       
Python 3.6.2

❯ notmuch --version
notmuch 0.24.2

```

**Comment by yang-ling** [comment]
Same error here.

1. afew version: 1.1.1.dev17+g090b511
2. Python version: 3.6.2
3. Notmuch version: 0.25

**Comment by yang-ling** [comment]
I found that changing `tags = notmuch_settings.get_list('new', 'tags')` to `tags = list(notmuch_settings.get_list('new', 'tags'))` in `NotmuchSettings.py` can solve this problem.

**Comment by GuillaumeSeren** [comment]
Hey,
I have encountered this error too while using python-3.x,
I think we have some issue with that version, so until those are fixed I suggest sticking with python-2.x.

But as I see there is already a PR, so if we can fix that it is good.

**Comment by flokli** [comment]
fix using filters in #161.
